Misc/NEWS.d/next/Tests/2022-06-16-13-26-31.gh-issue-93018.wvNx76.rst

```diff
@@ -0,0 +1 @@
+Fix for the compatibility problems with expat made more robust.
```

Comment:
I'm not sure this is the best wording. How about:
`Make two tests forgiving towards host system libexpat with backported security fixes applied.`

Reply:
Done.

Reply:
Thank you!